guide/payments/request.md

```diff
@@ -0,0 +1,471 @@
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/reusable-QR.png"
+   retina = "/assets/images/guide/payments/request/reusable-QR@2x.png"
+   layout = "shadow"
+   caption = "Reusable payment request QR codes can be used in places that regularly receive payments like a donations page."
+   alt-text = "Screen showing reusable QR code."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/payment-link.png"
+   retina = "/assets/images/guide/payments/request/payment-link@2x.png"
+   layout = "shadow"
+   caption = "Payment links are encoded so they open directly in local bitcoin applications."
+   alt-text = "Screen showing payment link UI."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/contactless.png"
+   retina = "/assets/images/guide/payments/request/contactless@2x.png"
+   layout = "shadow"
+   caption = "Sharing a payment request contactless is a quick way to request a payment in-person."
+   alt-text = "Screen showing contactless UI."
+   width = 250
+   height = 541
+%}
+
+</div>
+
+## Receiving a payment
+
+Once a request has been shared, the sender then needs to send bitcoin to the request. We cover the design considerations and user flows for sending bitcoin in our [sending bitcoin]({{ "/guide/payments/send/" | relative_url }}) page.
+
+### Lightning
+
+Receiving a Lightning payment is something that involves a lot of background complexity. How a wallet application manages this, has a major impact on the end user's experience.
+
+To receive a Lightning payment, the receiver needs a payment channel with inbound liquidity and also has to be online. Some wallets offer services that allow channels to be opened on-demand if a user has no inbound liquidity when receiving an incoming payment. They may also hold payment for a user until they are online to forward it to them. These services require a wallet to be solely connected to the peer that offers these services. We cover what services are commonly offered and how they work on our Lightning services page.
+
+Without Lightning services, a user will need to obtain some inbound liquidity before receiving a Lightning payment. This can be done by opening a channel with a peer that offers inbound liquidity and/or sending payments moving outbound to inbound capacity. It is recommended to let users know if they do not have any inbound liquidity before they share a payment request to prevent a payment failure.
+
+### On-chain
+
+Receiving on-chain involves the sender broadcasting a transaction to the network for it to be confirmed. As on-chain transaction confirmation times vary based on network congestion and fees being paid, it is uncertain exactly when a transfer will be considered complete. Keep users informed of the state of their incoming transfers.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
```

Comment:
The title is "On-chain", but the images discuss Lightning liquidity?

Reply:
Those images are for the whole "Receiving a payment" section not just for the on-chain section. I'll make this more clear.